README.md

```diff
@@ -369,4 +381,4 @@ instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or
 
 ## Contributing
 
-[See CONTRIBUTING.md](./CONTRIBUTING.md)
+[See CONTRIBUTING.md](./CONTRIBUTING.md)
```

Comment:
hmm, I wonder what Github thinks has changed here...?

Reply:
I am not sure either! May be line number?